styles/components-styles/card.css

```diff
@@ -2,6 +2,12 @@
   border-radius: 0.5rem;
   background: white;
   position: relative;
+  border: 1px solid var(--COLOR-GREY);
+}
+
+.card:hover {
+  box-shadow: 0 3px 5px -1px rgb(0 0 0 / 20%), 0 2px 10px 0 rgb(0 0 0 / 14%), 0 0 18px 0 rgb(0 0 0 / 12%);
+  transition: all 0.3s ease;
```

Comment:
Instead of using `all` can use `box-shadow` cause only box-shadow is changing.
And you can move the transition to the `.card` if you want the transition on both while entering in hover state and while leaving it.

Reply:
yes, moved transition to `.card` for `box-shadow`

Reply:
great